.github/PULL_REQUEST_TEMPLATE

```diff
@@ -13,4 +13,7 @@ Please make sure your commit message summary line and pull request
 title match the Go convention; a one-line summary of the change,
 prefixed by the primary affected package that should complete the
 sentence, "This change modifies Gonum to _____."
+
+Do not delete any of this templated text in the PR description, but
+feel free to add addition context.
```

Comment:
"additional" or omit?

Reply:
Sorry, I meant there's a typo, should be "additional", not "addition".

Reply:
:facepalm: Thank you. I need get English lessons from you as well as Czech.

Reply:
I see it so often that people write about adding additional something and the redundancy has always bothered me and from now it won't :tada: :v: